cmd/mattermost/commands/channel.go

```diff
@@ -562,7 +562,7 @@ func modifyChannelCmdF(command *cobra.Command, args []string) error {
 
 	user := getUserFromUserArg(a, username)
 	if user == nil {
-		return fmt.Errorf("Unable to find user: '%v'", username)
+		return fmt.Errorf("unable to find user: '%v'", username)
```

Comment:
The cmd/mattermost packages are a bit special. The error returned is directly displayed in the console. So we have to keep the error strings as-is because there is nothing prefixing them with a proper user facing string.

Please revert all changes in cmd/mattermost.

Reply:
fixed